src/reflow/mod.rs

```diff
@@ -447,12 +445,86 @@ should be rewrapped."#;
 
     #[test]
     fn reflow_markdown_two_paragraphs() {
-        reflow!(60 break ["Possible **ways** to run __rustc__ and request various parts of LTO.",
-                          " ",
-                          "Some more text after the rule which represents a paragraph"] =>
+        const CONTENT: &'static str =
+            " /// Possible **ways** to run __rustc__ and request various parts of LTO.
+///
+/// Some more text after the rule which represents a paragraph";
+
+        let expected = vec![
             r#" Possible **ways** to run __rustc__ and request various
- parts of LTO.
+ parts of LTO."#,
+            r#" Some more text after the rule which represents a
+ paragraph"#,
+        ];
+
```

Comment:
The reflow check here is correct, but we could never use this as a direct replacement, since it would cover additional lines.
We would need a injection suggestion with a source span length of 0 (so nothing is removed, only text is inserted at that position). If this is the case, the comment type (`///` in this case ) needs to be added as a injection for any additional lines inserted.